db/schema.sql

```diff
@@ -6,10 +6,10 @@ USE appointments;
 
 CREATE TABLE users (
   id INT NOT NULL AUTO_INCREMENT,
-  first_name VARCHAR(20),
-  last_name VARCHAR(20),
-  phone INT(10),
-  email VARCHAR(20),
+  first_name VARCHAR(50),
+  last_name VARCHAR(50),
+  phone VARCHAR(20),
```

Comment:
lol i too thought phone number were int.

Reply:
It's so mean sometimes.